llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -4164,8 +4164,6 @@ class VPlan {
   SmallVector<VPValue *, 16> VPLiveIns;
 
   /// Mapping from SCEVs to the VPValues representing their expansions.
-  /// NOTE: This mapping is temporary and will be removed once all users have
```

Comment:
This is an interesting one, originally I was a bit surprised and while trying to confirm if the TODO is still needed it turns out the while map isn't needed any longer: https://github.com/llvm/llvm-project/pull/164490/files